sources/ClangSharp/Extensions/CXTranslationUnit.cs

```diff
@@ -32,45 +88,100 @@ public partial struct CXTranslationUnit : IDisposable
 
         public CXTargetInfo TargetInfo => clang.getTranslationUnitTargetInfo(this);
 
-        public void AnnotateTokens(CXToken[] tokens, CXCursor[] cursors) => clang.annotateTokens(this, tokens, (uint)tokens.Length, cursors);
-
-        public void Dispose() => clang.disposeTranslationUnit(this);
-
-        public void DisposeTokens(CXToken[] tokens) => clang.disposeTokens(this, tokens, (uint)tokens.Length);
+        public void AnnotateTokens(CXToken[] tokens, CXCursor[] cursors)
```

Comment:
It would be nice if most of these were updated to take a `ROSpan<T>` or `Span<T>`, as appropriate. So you can avoid allocations where appropriate, but I left that to a future PR.